data/json/items/magazine/545x39.json

```diff
@@ -43,8 +59,16 @@
     "//": "Compared to 30rd and used online references.",
     "looks_like": "stanag30",
     "type": "MAGAZINE",
-    "name": { "str": "AK-74M 60-round magazine" },
-    "description": "A 60-round polymer magazine compatible with the AK-74M rifle.",
+    "name": { "str": "5.45mm 60-round magazine" },
+    "description": "A 60-round polymer magazine compatible with 5.45x39mm rifles.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "casket74mag",
```

Comment:
These should just be obsoleted. PUFGUN knock-off mags are kinda available, but they're knock offs of already unreliable magazines.

The real ones are unobtainium.

Reply:
> These should just be obsoleted. PUFGUN knock-off mags are kinda available, but they're knock offs of already unreliable magazines.
> 
> The real ones are unobtainium.

I’ll poke at the descriptions and remove the 60-round magazine; from what I can see, it doesn’t even look like it has a spawn entry, unless I’m overlooking something. GIT is currently screaming that it can't’ pull or push commits, though, so I need to figure out how to make it happy first. Either that, or I close and re-open this PR.

Reply:
It's an artisan gun thing. suggestion retracted, it's fine if you get the 1 casket mag in your state because of a quest.